src/js/overlay.js

```diff
@@ -116,7 +111,11 @@ const Overlay = function(id, opts) {
 		this.opts.trigger.addEventListener('click', triggerClickHandler.bind(this.opts.trigger, id), false);
 		this.context = this.opts.arrow ? oLayers.getLayerContext(this.opts.arrow.target) : oLayers.getLayerContext(this.opts.trigger);
 	} else {
-		this.context = this.opts.arrow ? oLayers.getLayerContext(this.opts.arrow.target) : document.body;
+		if (this.opts.parentNode) {
+			this.context = document.querySelector(this.opts.parentNode);
```

Comment:
`querySelector` can return `null` if it does not find a match in the DOM. Should we throw an error or set the context to `document.body`?